.github/workflows/post-merge.yml

```diff
@@ -323,6 +340,52 @@ jobs:
           echo "Building the code"
           make build
 
+      - name: Run extraction
+        if: ${{ inputs.run_extraction}}
+        working-directory: ${{ inputs.project_folder }}
+        shell: bash
+        run: |
+         echo "extract packages"
+         make extraction
+
+      - name: Upload Assets
+        if: ${{ inputs.run_upload_assets}}
+        working-directory: ${{ inputs.project_folder }}
+        env:
+          EXTRACTION_DIR: "${{ inputs.extraction_dir }}"
+        run: |
+          sudo apt update
+          sudo apt install -y software-properties-common
+          curl -fsSL https://cli.github.com/packages/githubcli-archive-keyring.gpg | sudo tee /usr/share/keyrings/githubcli-archive-keyring.gpg > /dev/null
+          echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/githubcli-archive-keyring.gpg] https://cli.github.com/packages stable main" | sudo tee /etc/apt/sources.list.d/github-cli.list > /dev/null
+          sudo apt update
+          sudo apt install gh
+
+          # Set environment variables
+          export GITHUB_TOKEN=${{ secrets.SYS_ORCH_GITHUB }}
```

Comment:
Use the same approach as version-tag. set/unset the token as actions/checkout does